sync/listen.py

```diff
@@ -16,51 +15,26 @@
 logger = log.get_logger(__name__)
 
 
-class Consumer(object):
-    """Represents a Pulse consumer to version control data."""
-    def __init__(self, conn, exchanges, extra_data):
-        self._conn = conn
-        self._consumer = None
-        self._entered = False
+class Consumer(ConsumerMixin):
```

Comment:
So, calling this Consumer, but having it be different from the `Consumer` we create in `get_consumers` is a little confusing.